### PR TITLE
Prepare for release 2.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**
I started with the changelog. I'm not very familiar with nodejs, could someone please take a look if I missed anything? I didn't find anything that looks like a major change, so I propose 1.4.0 as the next version.

### CODE REVIEW

**Comment by bjoernhaeuser** [comment]
When using current master (818f81e0dfdf9ebf251600c3bb9513b336b34206), I get these messages:

```
# puppet agent --test
Notice: Ignoring --listen on onetime run
Info: Retrieving pluginfacts
Info: Retrieving plugin
Info: Loading facts
Error: Could not autoload puppet/provider/package/npm: /var/lib/puppet/lib/puppet/provider/package/npm.rb:3: syntax error, unexpected ':', expecting $end
...package).provide :npm, parent: Puppet::Provider::Package do
                              ^
Error: Could not retrieve local facts: Could not autoload puppet/provider/package/npm: /var/lib/puppet/lib/puppet/provider/package/npm.rb:3: syntax error, unexpected ':', expecting $end
...package).provide :npm, parent: Puppet::Provider::Package do
                              ^
Error: Failed to apply catalog: Could not retrieve local facts: Could not autoload puppet/provider/package/npm: /var/lib/puppet/lib/puppet/provider/package/npm.rb:3: syntax error, unexpected ':', expecting $end
...package).provide :npm, parent: Puppet::Provider::Package do
                              ^
```

Any idea where this is coming from?

**Comment by bastelfreak** [comment]
Are you on ruby1.8? We don't support that anymore.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+- allow absent for ensure attribute
+
+### Fixed
+
+- Link to badges
+- Correct handle of ensure attribute for nodejs
+
+### Maintenance
+
+- partial reformat of code for better readability
```
Comment (bastelfreak) [inline CHANGELOG.md]: I would bump the module to 2.0.0 because this is a breaking change. I added a note about ruby to the `Maintenance` part.

**Comment by juniorsysadmin** [comment]
Looks alright to me

**Comment by jyaworski** [comment]
Rubocop.

**Pull request merged**